lib/abr/simple_abr_manager.js

```diff
@@ -333,7 +380,7 @@ shaka.abr.SimpleAbrManager = class {
 
         return shaka.util.StreamUtils.meetsRestrictions(
             variant, restrictions,
-            /* maxHwRes= */ {width: Infinity, height: Infinity});
+            /* maxHwRes= */ {width: maxWidth, height: maxHeight});
```

Comment:
`shaka.util.StreamUtils.meetsRestrictions` already filters on `restrictions.maxWidth` and `restrictions.maxHeight`, why add new args?

Reply:
See the previous thread